packages/nextra/styles/steps.css

```diff
@@ -1,11 +1,16 @@
-.nextra-steps h3 {
-  counter-increment: var(--counter-id);
-
-  &:before {
-    @apply _absolute _size-[33px];
-    @apply _border-4 _border-[rgb(var(--nextra-bg))] _bg-gray-100 dark:_bg-neutral-800;
-    @apply _rounded-full _text-neutral-400 _text-base _font-normal _text-center _-indent-px;
-    @apply _mt-[3px] _ml-[-41px];
-    content: counter(var(--counter-id));
+.nextra-steps {
+  h2,
+  h3,
+  h4 {
+    counter-increment: var(--counter-id);
+    @apply _relative _mt-8 _border-none _text-xl _font-semibold;
```

Comment:
I referred to the current font size of the theme blog, which prevents the border-left of the steps from exceeding the count badge. 

![image](https://github.com/user-attachments/assets/685978f9-f045-446f-ba3d-9f1cbb338161)

Reply:
I think it was intentionnaly per design